enterprise/api/v1/serializers.py

```diff
@@ -214,3 +215,88 @@ def update(self, instance, validated_data):
     user = UserSerializer(read_only=True)
     enterprise_customer = EnterpriseCustomerSerializer(read_only=True)
     data_sharing_consent = UserDataSharingConsentAuditSerializer(many=True, read_only=True)
+
+
+class EnterpriseCatalogSerializer(serializers.Serializer):
+    """
+    Serializer for enterprise customer catalog.
+    """
+    # pylint: disable=invalid-name
+    id = serializers.IntegerField(read_only=True, help_text=_('Course catalog primary key.'))
+    name = serializers.CharField(help_text=_('Catalog name'))
+    query = serializers.CharField(help_text=_('Query to retrieve catalog contents'))
+    courses_count = serializers.IntegerField(read_only=True, help_text=_('Number of courses contained in this catalog'))
+    viewers = serializers.ListField(
+        allow_null=True, allow_empty=True, required=False,
+        help_text=_('Usernames of users with explicit access to view this catalog'),
+        style={'base_template': 'input.html'},
+        child=serializers.CharField(),
+    )
+
+    def create(self, validated_data):
+        """
+        Do not perform any operations for state changing requests.
+        """
+        pass
+
+    def update(self, instance, validated_data):
+        """
+        Do not perform any operations for state changing requests.
+        """
+        pass
+
+
+class EnterpriseCatalogCoursesSerializer(serializers.Serializer):
+    """
+    Serializer for enterprise customer catalog courses.
+    """
+    count = serializers.IntegerField(read_only=True, help_text=_('Total count of catalog courses.'))
+    next = serializers.CharField(read_only=True, help_text=_("URL to fetch next page of courses."))
```

Comment:
yes, but we can not change it to something as this field is expected in catalog api response.